adapters/sn2core/sn2core.go

```diff
@@ -262,12 +272,16 @@ func AdaptDeployAccountTransaction(t *starknet.Transaction) *core.DeployAccountT
 	}
 }
 
-func AdaptCairo1Class(response *starknet.SierraDefinition, compiledClass *starknet.CompiledClass) (*core.Cairo1Class, error) {
+func AdaptCairo1Class(
+	response *starknet.SierraDefinition,
+	compiledClass *starknet.CompiledClass,
+) (*core.Cairo1Class, error) {
 	var err error
 
 	// TODO: what's the absolute minimum size of a Sierra Definition?
 	// A Sierra program size should be at least 3 to contain the version or 1 if it's version is 0.1.0
-	if len(response.Program) < 3 && (len(response.Program) == 0 || !response.Program[0].Equal(&core.SierraVersion010)) {
+	if len(response.Program) < 3 &&
+		(len(response.Program) == 0 || !response.Program[0].Equal(&core.SierraVersion010)) {
```

Comment:
Personal opinion: I think we should rewrite this line, because we should avoid breaking line right before indentation change ([source](https://google.github.io/styleguide/go/guide#line-length))